### PR TITLE
Added setup and run scripts (GiveMeBagdeWINDOWS.bat and GiveMeBadgeMAC_LINUX.sh)

### PR DESCRIPTION
**Pull request by Buxx0**
Scripts that automatically install the requirements, and then run the bot.

### CODE REVIEW

File: GiveMeBadgeMAC_LINUX.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+set +v
+echo echo Installing requirements for givemebadge...
```
Comment (alexanderdavidj) [inline GiveMeBadgeMAC_LINUX.sh:2]: `echo echo` misspell. It should be just one `echo`

**Review by alexanderdavidj — CHANGES_REQUESTED** [review]

File: GiveMeBadgeMAC_LINUX.sh (hunk 1 above)
Comment (Buxx0) [reply GiveMeBadgeMAC_LINUX.sh]: Fixed.

**Review by alexanderdavidj — APPROVED** [review]

**Comment by ZackeryRSmith** [comment]
An idea for the file names: `setup.bat` and `setup.sh`. I find `GiveMeBagdeWINDOWS.bat` and `GiveMeBadgeMAC_LINUX.sh` to be a bit wordy. Just my personal opinion, though.

**Comment by AlexFlipnote** [comment]
There is only need to have a starting script that can also tell you what is wrong if anything. I have added such to the script.
